insights/parsers/slabinfo.py

```diff
@@ -0,0 +1,104 @@
+"""
```

Comment:
@vishwanathjadhav please add a file in `docs/shared_parsers_catalog` to include this parser in the documentation.  See the [documentation guidelines](https://insights-core.readthedocs.io/en/latest/docs_guidelines.html) for more information.

Reply:
Done